Makefile

```diff
@@ -103,7 +103,15 @@ min_debug:				#: Minimal build with debugging symbols
 	$(MAKE) build BUILD_DIR=debug
 
 benchmarks-basic-build:
-	$(MAKE) release EXTRA_CMAKE_FLAGS="${EXTRA_CMAKE_FLAGS} -DVELOX_ENABLE_BENCHMARKS_BASIC=ON"
+	$(MAKE) release EXTRA_CMAKE_FLAGS=" ${EXTRA_CMAKE_FLAGS} \
+                                            -DVELOX_BUILD_TESTING=OFF \
+                                            -DVELOX_ENABLE_BENCHMARKS_BASIC=ON"
+
+benchmarks-build:
+	$(MAKE) release EXTRA_CMAKE_FLAGS=" ${EXTRA_CMAKE_FLAGS} \
+                                            -DVELOX_ENABLE_BENCHMARKS=ON \
+                                            -DVELOX_ENABLE_BENCHMARKS_BASIC=ON \
```

Comment:
Do we need to set ENABLE_BENCHMARKS_BASIC if ENABLE_BENCHMARKS is ON, the latter should ensure that Basic is set to ON right?

Reply:
`ENABLE_BENCHMARKS` did not enable `ENABLE_BENCHMARKS_BASIC` before. I fixed it now.